libraries/joomla/form/domfieldinterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @package     Joomla.Platform
+ * @subpackage  Form
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE
+ */
+defined('JPATH_PLATFORM') or die();
+
+/**
+ * Interface which marks a JFormField as available for com_fields.
+ *
+ * @since 3.7
+ */
+interface JFormDomFieldInterface
```

Comment:
This should either be called `JFormDomfieldinterface` or moved to `libraries/joomla/form/dom/field/interface.php` so that the autoloader will correctly load it.  Both cases remove the need for https://github.com/joomla-projects/custom-fields/pull/104/files#diff-9d799c43096f42c965b28c44a8d8a8b1R12